grails-app/services/org/pih/warehouse/inventory/InventorySnapshotService.groovy

```diff
@@ -699,7 +699,7 @@ class InventorySnapshotService {
 
     def getTransactionReportDetails(Location location, List<Category> categories, List<Tag> tags, List<ProductCatalog> catalogs, Date startDate, Date endDate) {
 
-        def transactionData = getTransactionReportData(location, startDate, endDate)
+        def transactionData = getTransactionReportData(location, startDate, endDate - 1)
```

Comment:
If you go to the getTransactionReportData, there is:
`transactionDateKey {
                between("date", startDate, endDate+1)
            }`
How about removing the `+1` there instead of doing the `- 1` here (and in the line below)?

Reply:
@awalkowiak Thanks, that's right!